portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -559,6 +559,92 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test asserts that user can filter app by Category in marketplace store."
+	@priority = "3"
+	test CanFilterMarketplaceStoreAppsByCategories {
+		property test.name.skip.portal.instance = "AppManagement#CanFilterMarketplaceStoreAppsByCategories";
+
+		task ("Given user login marketplace") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+		}
+
+		task ("When user filter by Category with value Page Layouts") {
+
+			// Navigate the store again for filter price failure issue
```

Comment:
Make sure this step is necessary and remember to update the description accordingly.

Reply:
DONE

Reply:
I tried we mast use the step